opencv-glib/image.cpp

```diff
@@ -23,6 +23,10 @@ typedef struct {
   gint thickness;
   GCVLineType line_type;
   gint shift;
+  gdouble tip_length;
+  gboolean bottom_left_origin;
```

Comment:
How about using `use_bottom_left_origin` to indicate this value is boolean explicitly.
`bottom_left_origin` may be misunderstood as origin (the pair of X point and Y point).

Reply:
I agree that `use_bottom_left_origin` is better name, but I think the consistency with C++/Python API is also important. Do you think which policy is better?
a. Glib API may use different name if C++ API's name is not so good.
b. Use same name as C++ API as much as possible.
For me, b is better because I don't have to think about better names. I can just use same name as C++/Python API.

Reply:
I like a.
If we use GLib friendly naming, GObject Introspection based bindings (e.g.: red-opencv) can get easy to use API. For example, If we use `cv_XXX_to_string()` method in opencv-glib, gobject-introspection gem generates `CV::XXX#to_s`.

> I don't have to think about better names.

We can think about better names together. :-)